index.ts

```diff
@@ -668,16 +668,22 @@ export class EthereumBlock {
  */
 export class EthereumTransaction {
   hash: Bytes
-  blockHash: Bytes
-  blockNumber: BigInt
+  index: BigInt
+  from: Address
+  to: Address
```

Comment:
Wherever we have an optional value in Rust / rust-web3, we should do `T | null` here (with `T` being the type of the value if it is set.

Reply:
Done (pushed to this PR).